test/helpers/cmd.go

```diff
@@ -279,6 +279,14 @@ func (res *CmdRes) WaitUntilMatchRegexp(expr string) error {
 		&TimeoutConfig{Timeout: HelperTimeout})
 }
 
+// GetErr returns error created from program output if command is not successful
+func (res *CmdRes) GetErr() error {
```

Comment:
Is this method strictly necessary? I don't really liked, if we use this method we'll end in the helpers functions writing something like this: 
```
if err:= res.GetErr(); err != nil  {
  return err
}
```
I think if we can avoid this one it'll be great, if you need to write `fmt.Errorf()` each time that something fails, you'll end writing a custom error for that, so you'll provide more context to the function. 

For asserts, I'm not worried at all, because the use of `res.ExpectSucess()` is largely used. But this is a personal opinion, so feel free to keep it.

Reply:
No, in the helpers no asserts should be done. The idea is to not have the helper that returns an error because if you have, that ends that no proper error messages will be written.

Reply:
How about adding a command and execution context to error?

Reply:
But that command output will not return any context, only that a command was not executed correctly and the command information, not enough, IMHO.

Reply:
I added context message to GetErr, so that error cosists of provided context and command output. Should be enough, right?